src/Module.Server/Rewards/CrpgRewardClient.cs

```diff
@@ -60,6 +60,16 @@ private void HandleRewardUser(CrpgRewardUser message)
                     color));
         }
 
+        if (message.Compensation != 0)
+        {
+            (Color color, string verb, string teamhitMessage) = message.Compensation > 0
+                ? (new Color(65, 105, 225), "Gained", "being teamhit")
+                : (new Color(0.74f, 0.28f, 0.01f), "Lost", "teamhitting others");
+            InformationManager.DisplayMessage(
+                new InformationMessage($"{verb} {message.Compensation} gold as compensation for {teamhitMessage}.",
+                    color));
+        }
+

```

Comment:
The thing is , contrary to gold reward and experience reward , this can be both negative and positive. I really think that having them seperated makes more senses in term of UX .